skimage/io/tests/test_mpl_imshow.py

```diff
@@ -31,6 +35,13 @@ def _reset_plugins():
     r"The figure layout has changed to tight|\A\Z",
 ]
 
+# This warning comes from the Python 3.12.1 interpreter powered by Pyodide
+# and is not relevant to the tests where it is raised.
```

Comment:
So this is a warning that has to be addressed upstream?

Reply:
Yes – this is most likely coming from https://github.com/python/cpython/issues/101588. Pyodide won't be able to address this until it bumps to CPython 3.13, which will happen in the next minor release (0.27.X) – the CPython version is coupled with the Pyodide version (and as is the Emscripten toolchain version).